.buildkite/scripts/setenv.sh

```diff
@@ -2,19 +2,28 @@
 
 set -euo pipefail
 
+#source .buildkite/scripts/common.sh
+
+WORKSPACE=${WORKSPACE:-"$(pwd)"}
+GO_VERSION=$(cat .go-version)
+
 export REPO="beats"
 export DOCKER_REGISTRY="docker.elastic.co"
 export SETUP_GVM_VERSION="v0.5.1"
 export DOCKER_COMPOSE_VERSION="1.21.0"
 export DOCKER_COMPOSE_VERSION_AARCH64="v2.21.0"
+
 export ASDF_NODEJS_VERSION="18.17.1"
 export AWS_REGION="eu-central-1"
+export KIND_VERSION="v0.20.0"
 
-WORKSPACE=${WORKSPACE:-"$(pwd)"}
 export WORKSPACE
-GO_VERSION=$(cat .go-version)
 export GO_VERSION
+export KUBECONFIG="${WORKSPACE}/kubecfg"
+export BIN="${WORKSPACE}/bin"
```

Comment:
The intention is to eventually remove this file.
With that in mind, and with the suggestions above, what if we remove the above env vars from here and instead add them in `.buildkite/deploy/kubernetes/scripts/setup-k8s-env.sh`, since anyway, they are k8s specific?

Note: you'll need to `source .buildkite/deploy/kubernetes/scripts/setup-k8s-env.sh` in the command: rather than just invoke it, so that the env vars remain available for the next command.